src/EcomProvider.cs

```diff
@@ -280,26 +280,9 @@ public EcomProvider(string connectionString)
         CreateMissingGoups = true;
     }
 
-    public override Schema GetOriginalSourceSchema()
+    public override Schema GetOriginalDestinationSchema()
```

Comment:
Now all virtual columns are moved to this method but look carefully some of them also are used in the EcomSourceReader:

FieldValueProductNumber
EcomStockUnit.ProductName
EcomStockUnit.StockLocationName